### PR TITLE
Add an announcement to our release process

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Note that these rules do not apply to the
 
 To create a new release:
 * Create a PR that modifies the [README](README.md), and all specifications
-  (ie. *.md files) that include a version string, to the new release
+  (ie. \*.md files) that include a version string, to the new release
```
Comment (duglin) [inline GOVERNANCE.md]: This is escaped to avoid github showing the rest of the file in italics.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -115,4 +115,9 @@ To create a new release:
     `git log --pretty=format:%s master...v0.1`.
     Just replace "v0.1" with the name of the previous release.
   * Press `Publish release` button
+* Create an "announcement" highlighting the key features of the new release
+  and any potential noteworthy activities of the group:
+  * Send it to the mailing list
+  * Add it to the "announcement" section of our
+    [website](https://cloudevents.io/)
```
Comment (duglin) [inline GOVERNANCE.md]: I'll update this link to the announcement section on it's created - @barkerd427 is working on this.

**Comment by duglin** [comment]
Conditionally approved on the Jan 17 call - once we add a reference to our twitter account.

Done - can I get an LGTM to make sure I did it right?

**Comment by cneijenhuis** [comment]
LGTM - the account could use a bit of love though, e.g. (re)tweet the 0.2 release.

**Comment by duglin** [comment]
Thanks - I'll work on getting a tweet....

**Pull request merged**